### PR TITLE
add logger helper class

### PR DESCRIPTION
**Pull request by djih**
tested on Android demo, disabling works to silence log statements

### CODE REVIEW

File: src/com/amplitude/api/AmplitudeLog.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        if (enableLogging) return Log.w(tag, tr);
+        return 0;
+    }
+
+    int w(String tag, String msg, Throwable tr) {
+        if (enableLogging) return Log.w(tag, msg, tr);
+        return 0;
+    }
+
+    int wtf(String tag, String msg) {
```
Comment (curtisliu) [inline src/com/amplitude/api/AmplitudeLog.java]: wtf

File: src/com/amplitude/api/AmplitudeLog.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    protected static AmplitudeLog instance = new AmplitudeLog();
+
+    public static AmplitudeLog getLogger() {
+        return instance;
+    }
+
+    private AmplitudeLog() {
+    } // prevent instantiation
+
+    void setEnableLogging(boolean enableLogging) {
```
Comment (curtisliu) [inline src/com/amplitude/api/AmplitudeLog.java]: would prefer this to be a setLogLevel

Comment (curtisliu) [inline src/com/amplitude/api/AmplitudeLog.java]: actually leave this and add a setLogLevel

File: src/com/amplitude/api/AmplitudeLog.java (hunk 1 above)
Comment (djih) [inline src/com/amplitude/api/AmplitudeLog.java]: lol are you saying wtf to the wtf method. That's a legit method: http://developer.android.com/reference/android/util/Log.html#wtf(java.lang.String, java.lang.Throwable)   it stands for What a Terrible Failure

Comment (curtisliu) [inline src/com/amplitude/api/AmplitudeLog.java]: hah

File: src/com/amplitude/api/AmplitudeLog.java (hunk 2 above)
Comment (djih) [inline src/com/amplitude/api/AmplitudeLog.java]: what is setLogLevel? That's not supported by the Android logger

**Comment by djih** [comment]
Added logLevel PTAL @curtisliu

**Comment by curtisliu** [comment]
LGTM

**Pull request merged**